File: CS/1684_Count_the_Number_of_Consistent_Strings.cs
public class Solution {
    public int CountConsistentStrings(string allowed, string[] words) {
        int result = words.Length;
        foreach(string word in words){
            foreach(char c in word){
                if(!allowed.Contains(c)){
                    result--;
                    break;
                }
            }
        }
        return result;
    }
}

File: Cpp/1684_Count_the_Number_of_Consistent_Strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        int result = words.size();
        for(string word : words){
            for(char c : word){
                if(allowed.find(c) == -1){
                    result--;
                    break;
                }
            }
        }
        return result;
    }
};
